sys-process/acct/files/acct.initd

```diff
@@ -11,10 +11,7 @@ depend() {
 
 checkconfig() {
 	export ACCT_LOG=${ACCT_LOG:-/var/account/pacct}
-	if [ ! -e "${ACCT_LOG}" ] ; then
-		touch "${ACCT_LOG}"
-		chmod 600 "${ACCT_LOG}"
-	fi
+	checkpath -f -m 600 "${ACCT_LOG}"
```

Comment:
You're updating the init file, but how can you ensure that the updated file arrives on users' machines? You'll need to revbump the latest stable and unstable ebuilds.